### ISSUE TITLE
`RqlDriverError: Object field 'originUrl' may not be undefined`

### ISSUE THREAD

**Issue by vjpr** [staygrimm/passwordless-rethinkdbstore#1]
I just hooked it up into the sample project and I get this error:

```
RqlDriverError: Object field 'originUrl' may not be undefined
    at new RqlDriverError (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/errors.js:14:13)
    at new MakeObject (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/ast.js:1527:15)
    at Function.<anonymous> (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/ast.js:3906:12)
    at Function.<anonymous> (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/util.js:39:16)
    at Table.<anonymous> (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/ast.js:1073:45)
    at Table.<anonymous> (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/util.js:60:16)
    at XXX/node_modules/passwordless-rethinkdbstore/lib/index.js:101:40
    at tryCatcher (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/node_modules/bluebird/js/main/util.js:24:31)
    at Promise.successAdapter (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/node_modules/bluebird/js/main/nodeify.js:22:30)
    at Promise._settlePromiseAt (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/node_modules/bluebird/js/main/promise.js:528:21)
    at Promise._settlePromises (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/node_modules/bluebird/js/main/promise.js:646:14)
    at Async._drainQueue (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/node_modules/bluebird/js/main/async.js:177:16)
    at Async._drainQueues (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/node_modules/bluebird/js/main/async.js:187:10)
    at Immediate.Async.drainQueues [as _onImmediate] (XXX/node_modules/passwordless-rethinkdbstore/node_modules/rethinkdb/node_modules/bluebird/js/main/async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:361:17)
```

Any ideas?

**Comment by vjpr** [comment]
Fix: 

index.js#96 Use `originUrl: originUrl || null` instead.

**Comment by fasiha** [comment]
Thanks for this, friends. Can we push an update to npm?

**Comment by staygrimm** [comment]
@fasiha I just published v1.0.2.  Thanks for the heads up!
